mtp_api/apps/transaction/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db import models
+
+from model_utils.models import TimeStampedModel
+
+from prison.models import Prison
+
+
+class Transaction(TimeStampedModel):
+    upload_counter = models.PositiveIntegerField()
+
+    prison = models.ForeignKey(Prison, blank=True, null=True)
+
+    prisoner_number = models.CharField(blank=True, max_length=250)
+    prisoner_name = models.CharField(blank=True, max_length=250)
```

Comment:
I think we should assume, for now, that we won't have prisoner name (and remove all the code related to it).

For some prisoners, it might be classified information that not even the business hub staff are allowed to see, so our system should probably not show it till we better understand all the rules around that.